Mesh_3/doc/Mesh_3/PackageDescription.txt

```diff
@@ -70,7 +70,6 @@ related to the template parameters of some models of the main concepts:
 
 - `BisectionGeometricTraits_3`
 - `IntersectionGeometricTraits_3`
-- `LabeledImage_3`
```

Comment:
Also remove the three deprecated classes from the ## classes ## section?

Reply:
I really wanted to bury `LabeledImage_3` because it was useless and buggy. But the deprecated classes are still documented, and I would like to keep them in the index of documented classes.

Is that an error?

Reply:
I thought they had to be removed from the reference manual main page. I'm actually rather in favor of keeping them as it's a smoother transition, so I'm all in favor of being wrong.